CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Logo
 - Memoization of class methods
 - Support for instances created with `Class#allocate`
+- Official testing and benchmarks for Ruby 3.0
```

Comment:
Worth noting that the code itself was also upgraded to have a ruby version of 3.0?

Reply:
Do you mean the `.ruby-version` file? I'm not sure that's useful, since that's only used for local development and not users of the gem or our automated tests. Or am I misunderstanding?

Reply:
No you're totally right, good point, it's internal to the gem.